README.md

```diff
@@ -58,6 +58,7 @@ npm install --save-dev ts-essentials
 ### Basic
 
 - `Primitive` type matching all primitive values.
+- `noop` function that takes any arguments and returns nothing. Makes a convenient default callback, Promise value eraser, etc.
```

Comment:
should this wording perhaps avoid the use of "etc" and just be 2 examples without leading on?

Reply:
TBH I'd avoid listing `Promise value eraser` as an example - you're the first person I've seen using this naming. A `noop` is a `noop` - a function that accepts anything, does nothing and returns nothing, just a convenient placeholder fn.